### PR TITLE
install guide for AUR

### PR DESCRIPTION
**Pull request by Ayyrik**
Please label before merging

### CODE REVIEW

File: docs/install/install-aur.md [hunk 1]
```diff
@@ -6,6 +6,34 @@ sidebar_position: 4
...
+
+You may use any terminal emulator of your choice.
+
+
+
+## Step 2: Install Vignette
+
+Please type `yay -S vignette-git` and press the `Enter` key. 
+
+![](/img/install/Konsole-yay.png)
```
Comment (sr229) [inline docs/install/install-aur.md:23]: You don't need this screenshot for no reason, use a codeblock.

File: docs/install/install-aur.md [hunk 2]
```diff
@@ -6,6 +6,34 @@ sidebar_position: 4
...
+
+You should select the right package, in this case by typing `1`.
+
+![](/img/install/yay-select.png)
+
+You may proceed with the installation by typing `y`.
+
+![](/img/install/yay-yes.png)
```
Comment (sr229) [inline docs/install/install-aur.md:31]: Explain only the selection of the right package. The person knows the terminal, they know how to say yes in a terminal.

**Review by sr229 — CHANGES_REQUESTED** [review]
Work on it more.

**Comment by Ayyrik** [comment]
i was told to write for idiots so thats what i did. also, yea, the pus is horrible but i didnt know how to make it smaller. asked adryzz

File: docs/install/install-aur.md [hunk 3]
```diff
@@ -6,6 +6,34 @@ sidebar_position: 4
 
 # Installing Vignette using AUR
 
-something
+If you wish to install Vignette on your Arch Linux computer, you may use any AUR helper of you choice. The package is [`vignette-git`](https://aur.archlinux.org/packages/vignette-git/). In the example bellow, i will be using yay.
 
--both-
```
Comment (adryzz) [inline docs/install/install-aur.md:11]: Why this? we need to remove the navigation on this page, so keep this.

**Review by adryzz — CHANGES_REQUESTED** [review]

**Comment by adryzz** [comment]
After i corrected everything you don't magically close it. we'll still work on this. Lemme get to my PC and I'll fix everything.

**Pull request closed without merging**